app/src/main/java/com/hedvig/app/feature/loggedin/ui/LoggedInFragmentViewModel.kt

```diff
@@ -0,0 +1,12 @@
+package com.hedvig.app.feature.loggedin.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+abstract class LoggedInFragmentViewModel : ViewModel() {
```

Comment:
This entire class can probably be a part of `LoggedInViewModel`. Also, we only need to use the `abstract`/`Impl`-pattern for `ViewModel`s when there is some behaviour that we want to be able to mock out - network calls, for example. This functionality should be the same regardless of if mocking is enabled or not, and can therefore stand on its own and needs no `abstract`/`Impl`-variant.